06_puppy/alfredxiao/mapstore.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/google/uuid"
+)
+
+type mapStore struct {
+	data map[string]Puppy
+}
+
+func (s *mapStore) CreatePuppy(p Puppy) string {
```

Comment:
Use [godoc](https://blog.golang.org/godoc-documenting-go-code) for all public functions.